nfc/CMakeLists.txt

```diff
@@ -18,13 +18,27 @@ endif()
 zephyr_include_directories(include)
 
 if(CONFIG_NFC_T2T_NRFXLIB)
-	zephyr_link_libraries(${NFC_LIB_PATH}/libnfct2t_nrf52.a)
+	if (CONFIG_SOC_SERIES_NRF52X)
+		zephyr_link_libraries(${NFC_LIB_PATH}/libnfct2t_nrf52.a)
+	endif()
+	if (CONFIG_SOC_SERIES_NRF53X)
+		zephyr_link_libraries(${NFC_LIB_PATH}/libnfct2t_nrf53.a)
+	endif()
 endif()
 
 if(CONFIG_NFC_T4T_NRFXLIB)
-	zephyr_link_libraries(${NFC_LIB_PATH}/libnfct4t_nrf52.a)
+	if (CONFIG_SOC_SERIES_NRF52X)
+		zephyr_link_libraries(${NFC_LIB_PATH}/libnfct4t_nrf52.a)
+	endif()
+	if (CONFIG_SOC_SERIES_NRF53X)
+		zephyr_link_libraries(${NFC_LIB_PATH}/libnfct4t_nrf53.a)
+	endif()
```

Comment:
```
if(CONFIG_NFC_T2T_NRFXLIB)
  zephyr_link_libraries(${NFC_LIB_PATH}/libnfct2t_${CONFIG_SOC_SERIES}.a)
endif()

```

Reply:
Done, thanks for the tip, it looks cleaner now.